server/test/bookmark.spec.js

```diff
@@ -0,0 +1,50 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../index';
+
+chai.use(chaiHttp);
+let userToken;
+
+describe('Bookmark routes', () => {
+  before(async () => {
+    const userResponse = await chai
+      .request(app)
+      .post('/api/v1/auth/login')
+      .send({
+        email: 'mofe@okoro.com',
+        password: 'soldier'
+      });
+
+    userToken = userResponse.body.data.token;
+  });
+
+  describe('POST /api/v1/', () => {
```

Comment:
Good job. Please complete the `describe` route here for easy readability.

Reply:
@samorahno Yes like @cupidcoder said, please make the `describe()` more descriptive for better readability

Reply:
Noted. I will complete the ```describe``` route